bundles/org.openhab.binding.liquidcheck/src/main/java/org/openhab/binding/liquidcheck/internal/LiquidCheckBindingConstants.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.liquidcheck.internal;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.openhab.core.thing.ThingTypeUID;
+
+/**
+ * The {@link LiquidCheckBindingConstants} class defines common constants, which are
+ * used across the whole binding.
+ *
+ * @author Marcel Goerentz - Initial contribution
+ */
+@NonNullByDefault
+public class LiquidCheckBindingConstants {
+
+    private static final String BINDING_ID = "liquidcheck";
+
+    // List of all Thing Type UIDs
+    public static final ThingTypeUID THING_TYPE_LIQUID_CHEK = new ThingTypeUID(BINDING_ID, "liquidCheckDevice");
+
+    // List of all Channel ids
+    public static final String CONTENT_CHANNEL = "content";
+    public static final String RAW_CONTENT_CHANNEL = "raw-content";
+    public static final String LEVEL_CHANNEL = "level";
+    public static final String RAW_LEVEL_CHANNEL = "raw-level";
+    public static final String FILL_INDICATOR_CHANNEL = "fill-indicator";
+    public static final String PUMP_TOTAL_RUNS_CHANNEL = "pump-runs";
+    public static final String PUMP_TOTAL_RUNTIME_CHANNEL = "pump-runtime";
+    public static final String MEASURE_CHANNEL = "measure";
+
+    // List of all Property ids
+    public static final String CONFIG_ID_FIRMWARE = "firmware";
+    public static final String CONFIG_ID_HARDWARE = "hardware";
+    public static final String CONFIG_ID_NAME = "name";
+    public static final String CONFIG_ID_MANUFACTURER = "manufacturer";
+    public static final String CONFIG_ID_UUID = "uuid";
+    public static final String CONFIG_ID_SECURITY_CODE = "securityCode";
+    public static final String CONFIG_ID_IP = "ip";
+    public static final String CONFIG_ID_HOSTNAME = "hostname";
+    public static final String CONFIG_ID_MAC = "mac";
+    public static final String CONFIG_ID_SSID = "ssid";
```

Comment:
Except for hostname, you should name your constants with PROPERTY_ rather than CONFIG_
Note that some properties are already defined in the Thing class and should be used if available:
```
public interface Thing extends Identifiable<ThingUID> {

    /** the key for the vendor property */
    final String PROPERTY_VENDOR = "vendor";

    /** the key for the model ID property */
    final String PROPERTY_MODEL_ID = "modelId";

    /** the key for the serial number property */
    final String PROPERTY_SERIAL_NUMBER = "serialNumber";

    /** the key for the hardware version property */
    final String PROPERTY_HARDWARE_VERSION = "hardwareVersion";

    /** the key for the firmware version property */
    final String PROPERTY_FIRMWARE_VERSION = "firmwareVersion";

    /** the key for the MAC address property */
    final String PROPERTY_MAC_ADDRESS = "macAddress";
```